astropy/table/column.py

```diff
@@ -1146,7 +1146,7 @@ def _encode_str(value):
 
     def tolist(self):
         if self.dtype.kind == "S":
-            return np.chararray.decode(self, encoding="utf-8").tolist()
+            return np.char.chararray.decode(self, encoding="utf-8").tolist()
```

Comment:
This may be painful for `io.fits` :scream:

Reply:
At least for now it seems `np.char` continues to exist. I think it is meant to go to a "legacy" grouping (see https://numpy.org/neps/nep-0052-python-api-cleanup.html#cleaning-up-the-submodule-structure). I had missed this, but fortunately @pllim just tested things and realized that it was just `np.chararray` that is the problem - it will be removed from the main namespace.

Reply:
Do you wanna think about future-proofing and remove usage of `np.char` altogether? 😬

Reply:
Maybe. Though I just noticed that there is a current PR over at numpy trying to speed it up by converting some of the code to C. So, maybe best to leave things as they are for now -- especially as we're dealing with so much churn already!